src/components/styles/NewBook.css

```diff
@@ -7,4 +7,77 @@ form {
 .input {
   padding: 10px;
   margin: 15px;
+} */
+
+.formSection {
+  display: block;
+  background-color: #fafafa;
+}
+
+/* .formDiv {
+  padding: 1rem 0;
+} */
+
+.footerDiv {
+  height: 0.125rem;
+  background-color: #e8e8e8;
+  width: 90%;
+  margin: 5%;
+}
+
+.formTitle {
+  font-size: 1.25rem;
+  font-weight: 700;
+  letter-spacing: -0.18px;
+  color: #888;
+  margin-left: 5%;
+}
```

Comment:
- [ ] Please 🙏🏽 correct the styles according to the font family specifications to match the Zeplin design.
![image](https://github.com/Atifzada/BookStore_App/assets/58642949/3ff1e522-36cb-4c1f-b33e-9428c7d4073b)

Reply:
Thank you for the helpful review. Surely I will fix these issue